project/DomainClassCreator.scala

```diff
@@ -738,6 +738,7 @@ object DomainClassCreator {
 
     println(s"writing results to $targetFile")
     outputFile.copyTo(targetFile, overwrite = true)
```

Comment:
```suggestion
    outputFile.moveTo(targetFile, overwrite = true)
```

Reply:
Unless you want the old file to be there if the process fails between the copy and the delete?